plugin.c

```diff
@@ -14,13 +14,13 @@ static int version(struct plugin *plugin)
 	struct program *prog = plugin->parent;
 
 	if (plugin->name) {
-		printf("%s %s version %s (git %s)\n",
```

Comment:
Hmm, I think this is ment to be the literal meaning 'version' not the function called version. So I don't think changing this part makes sense.

Reply:
let me fix this myself.

Reply:
Thanks for your comments for the tool warnings below let us change the function namings as `version`/`usage` to `version_cmd`/`usage_cmd` by the PR #1966.
```sh
tokunori@tokunori-desktop:~/nvme-cli$ git reset --hard linux-nvme/master
HEAD is now at 9077d042 fabrics: prevent 'nvme connect' to well-known discovery NQN
tokunori@tokunori-desktop:~/nvme-cli$ ~/centos-stream-9/scripts/checkpatch.pl --no-tree plugin.c
WARNING: Prefer using '"%s...", __func__' to using 'version', this function's name, in a string
#17: FILE: plugin.c:17:
+               printf("%s %s version %s (git %s)\n",

WARNING: Prefer using '"%s...", __func__' to using 'version', this function's name, in a string
#20: FILE: plugin.c:20:
+               printf("%s version %s (git %s)\n",

WARNING: Prefer using '"%s...", __func__' to using 'version', this function's name, in a string
#23: FILE: plugin.c:23:
+       printf("libnvme version %s (git %s)\n",

WARNING: Prefer using '"%s...", __func__' to using 'usage', this function's name, in a string
#63: FILE: plugin.c:63:
+               printf("usage: %s %s %s\n", prog->name, plugin->name, prog->usage);

WARNING: Prefer using '"%s...", __func__' to using 'usage', this function's name, in a string
#65: FILE: plugin.c:65:
+               printf("usage: %s %s\n", prog->name, prog->usage);

total: 0 errors, 5 warnings, 218 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

plugin.c has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.
```

Reply:
Alright, don't mind renaming these functions.